nemo/collections/vlm/neva/model/base.py

```diff
@@ -455,6 +459,7 @@ def forward(
         self,
         input_ids: torch.Tensor,
         position_ids: torch.Tensor,
+        image_sizes: List[torch.Tensor] = None,
```

Comment:
if we are going to change the signiture, it would be better we are using 2 model class? it will might break something we will use nemo / mcore llava

Reply:
It should not break anything for vanilla neva as image_sizes tensor is optional and will be used only for llava_next. The reason for using the neva model class is most of the functionality is same.